util/Setup/Templates/NginxConfig.hbs

```diff
@@ -120,6 +120,13 @@ server {
 
   location /api/ {
     proxy_pass http://api:5000/;
+
+      location /api/public/organization/import {
+        proxy_pass http://api:5000/public/organization/import;
```

Comment:
Not sure you have to set the proxy_pass on this location or not, just the timeout properties, but I could be wrong

Reply:
I wasn't sure either. I'll definitely defer to experience, but the info I found (e.g. https://blog.martinfjordvald.com/understanding-the-nginx-configuration-inheritance-model/) suggests nested locations overwrite all settings. I didn't find any examples similar to what we're trying to do, though.

I agree that it feels like nesting is useless if it just overwrites previous settings.

Reply:
If nested locations indeed do overwrite all settings, what are you thoughts about pulling out the new location to be on the same level as api; similar to how we are explicitly specifying the `app-id.json` endpoint outside of the `/` endpoint?

Reply:
I was worried in that case a call to `/api/public/organization/import` might be gobbled up by the `api` settings.